### PR TITLE
fix issue #398 (Python binding no deep copy of detail pointer)

### PR DESCRIPTION
**Pull request by w4kfu**
Issue and PoC: https://github.com/aquynh/capstone/issues/398

### CODE REVIEW

File: bindings/python/capstone/__init__.py [hunk 1]
```diff
@@ -441,6 +441,9 @@ def copy_ctypes(src):
     """Returns a new ctypes object which is a bitwise copy of an existing one"""
     dst = type(src)()
     ctypes.pointer(dst)[0] = src
+    if hasattr(src, "detail"):
```
Comment (aquynh) [inline bindings/python/capstone/__init__.py]: this line does not look right to me. i think it should be changed to something like:

    if self._cs._detail and self._raw.id != 0:

(you need to pass `_cs._detail` & `_raw.id` to `copy_ctypes()` to have this check)

**Comment by w4kfu** [comment]
Applied your idea from this comment: https://github.com/aquynh/capstone/issues/398#issuecomment-112860037.

I didn't use copy_ctypes() for detail copy, it's done directly in CsInsn constructor, and kept the test on \_cs.\_detail and \_raw.id.

**Comment by aquynh** [comment]
yes it looks good now. then we still need to remove the redundant `copy_ctypes()` function now, right?

**Comment by w4kfu** [comment]
Sorry, totally forgot to remove the copy of detail in `copy_ctypes()`

**Comment by aquynh** [comment]
merged, thanks!

**Pull request merged**